_posts/en/2020-10-27-cio-digital-transformation.md

```diff
@@ -0,0 +1,216 @@
+</style>
+
+In this blog, we suggest that Chief Information Officers (CIOs) and Chief Financial Officers (CFOs) are essential to bring a Digital Transformation vision into a reality.
+Although Digital Transformation deals a lot with changing business operations, if technology releases are not able to push through, that vision will remain a dream and will not manifest itself.
+We will show how critical CFOs are in allowing technology to respond faster to continuous business changes, and how Cloud and DevOps are opportunities that cannot be overlooked.
+
+<!--more-->
+
+_Note: This blog will link to internal ESDC documents, which are unfortunately only accessible within ESDC corporate network._
+
+## What Digital Transformation Entails
+
+On April 1, 2020, [Treasury Board's (TB) Policy on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32603) came into effect, which serves "_as an integrated set of rules that articulate how Government of Canada organizations manage service delivery, information and data, information technology, and cyber security in the digital era_."
+The Policy supports the [mandate of the Minister for Digital Government](https://pm.gc.ca/en/mandate-letters/2019/12/13/minister-digital-government-mandate-letter) and is guided by a commitment to the guiding principles and best practices of the [Government of Canada Digital Standards](https://www.canada.ca/en/government/system/digital-government/government-canada-digital-standards.html).
+
+As the Policy's requirements target Deputy Heads, we look at its associated [Directive](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601) for requirements targeting the senior officials designated to be responsible for leading particular functions (i.e., IM/IT, Service, and Cyber Security).
+Here are some statistics the IT Strategy team did ([link to source data](/assets/files/Policy_on_service_and_digital_analysis.xlsx)):
+
+<table>
+  <tr>
+    <th>Official</th>
+    <th>&#37; of Total Requirements</th>
+    <th>Mandatory Procedures (4) requirements</th>
+  </tr>
+  <tr>
+    <td style="font-size:x-large; font-weight:bold;">CIO <span style="font-size:medium">(w/ Chief Data Officer)</span></td>
+    <td style="font-size:x-large; font-weight:bold;">84&#37;</td>
+    <td style="font-size:x-large; font-weight:bold;">229</td>
+  </tr>
+  <tr>
+    <td>Service</td>
+    <td>10&#37;</td>
+    <td>0</td>
+  </tr>
+  <tr>
+    <td>Cyber Security</td>
+    <td>6&#37;</td>
+    <td>74</td>
+  </tr>
+</table>
+
+With such a focus on CIOs and CDOs, TB formally recognizes the **ubiquitous nature of technology in delivering services** to Canadians.
+Because CIOs and CDOs are accountable for such a large number of requirements, we claim they also need to be in a position to determine _how_ technology investments are to be managed (a domain historically attributed to CFOs).
+**Elected officials** now recognize that technology is at the core of service delivery.
```

Comment:
Do we infer that by the nature of the policy changes or by the mandate letter contents? (Just wondering if there's a quote we can make somewhere)